src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -1518,6 +1524,7 @@ public partial interface ITypeLib2 : System.Runtime.InteropServices.ComTypes.ITy
         void GetAllCustData(System.IntPtr pCustData);
         void GetCustData(ref System.Guid guid, out object pVarVal);
         new void GetDocumentation(int index, out string strName, out string strDocString, out int dwHelpContext, out string strHelpFile);
+        [System.Runtime.InteropServices.LCIDConversionAttribute(1)]
```

Comment:
This attribute is implementation detail. It does not make sense to have it in the reference assembly.

Reply:
E.g. we do not have `PreserveSig` on `ReleaseTLibAttr` either.

Reply:
Ok so then we should add this Attribute to the ignore list we have in corefx for GenAPI and ApiCompat probably. I will do that.

Reply:
>This attribute is implementation detail.

@jkotas I agree with this, but it also modified the corresponding native method signature. Are these ref assemblies ever viewed directly by consumers or simply used at build time? If they are used as a visual reference, it might make sense to keep around.

Reply:
The ref assemblies are not meant to be viewed directly by consumers.

The ref assemblies do not capture details relevant to interop. There are number of other things off, it is not just the attributes: methods are in the alphabetical order (the implementation order matters for interop), fields are missing (the struct layout is not preserved), ...

Reply:
Then, does StructLayoutAttribute matter in ref assemblies? Because these ref assembly has a lot of them.

Reply:
StructLayoutAttribute matters to C# compiler. It uses different definite assignment rules for struct with Explicit layout.

`StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)` attributes on structs are not necessary though. `LayoutKind.Sequential` is the default in C#, so explicitly specifying it is redundant.